c#/Mandoline.Api.Examples/Client.Repl/ConsoleOutput.cs

```diff
@@ -4,181 +4,253 @@
 // root for full license information.
 // </copyright>
 
-namespace Client.Repl
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Core;
+using Core.Client;
+using Core.Client.Models;
+using Core.Client.ServiceModels;
+
+namespace Client.Repl;
+
+// this implementation of Output directs output to console
+internal class ConsoleOutput : Output
 {
-    using System;
-    using System.Collections.Generic;
-    using System.Data;
-    using System.Linq;
-    using System.Text;
-    using System.Threading.Tasks;
-    using Core;
-    using Mandoline.Api.Client.Models;
-    using Mandoline.Api.Client.ServiceModels;
-
-    // this implementation of Output directs output to console
-    internal class ConsoleOutput : Output
-    {
-        public ConsoleOutput()
-        {
-        }
+    public ConsoleOutput()
+    {
+    }
 
-        // status isn't indicated in this client
-        private string StatusLabelText { get; set; }
+    // status isn't indicated in this client
+    private string StatusLabelText { get; set; }
 
-        // status isn't indicated in this client
-        public override void UpdateStatus(string v)
-        {
-        }
+    // status isn't indicated in this client
 
-        // status isn't indicated in this client
-        public override void UpdateStatus(bool v)
-        {
-        }
+    /// <inheritdoc/>
+    public override void UpdateStatus(string v)
+    {
+    }
+
+    // status isn't indicated in this client
+
+    /// <inheritdoc/>
```

Comment:
maybe we should suppress this, it appears in many places and don't think it is adding value